src/test/java/com/varchenko/FibonacciSequenceTest.java

```diff
@@ -0,0 +1,29 @@
+package com.varchenko;
+
+import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.api.Test;
+import static com.varchenko.FibonacciSequence.fib;
+
+public class FibonacciSequenceTest {
+    FibonacciSequence fibonacciSequence = new FibonacciSequence() ;
+        @Test
+        void Fibonacci_numbersFrom0to10() {
+            for (int index = 0; index < 10; index++) {
+
+                int expectedResult = fib(index);
+
+                int result = fib(index);
+                assertEquals(expectedResult, result);
+                System.out.println(fib(index));
+            }
+        }
+
+        @Test
+        void Fibonacci_SpecificNumber() {
+            int expectedResult = fib(10);
+            int result = 55;
+
+            assertEquals(expectedResult, result);
+            System.out.println(fib(10));
```

Comment:
That's a good one. 

But please, don't use `System.out.println` in the tests if you not trying to print something useful. 
As a programmer, I won't take any information from `55` written to the console. 

P.S. you have the value in the `expectedResult ` field. Why do you call the method again? You just wasting compute time and RAM.

 P.P.S. You confused the `expectedResult ` and `result ` fields. Switch them back.

Reply:
done